From bc4c49812bab6a57182ee7a3ecb6612811c19ffb Mon Sep 17 00:00:00 2001
From: wwidjaja <wwidjaja@maxlinear.com>
Date: Wed, 7 Apr 2021 16:16:32 +0800
Subject: [PATCH] PONRTSYS-8722 - add spinand MX35LF2G14AC 2Gbit

  add spinand MX35LF2G14AC 2Gbit to spinand_flash_ids[]

  NOTE: to be tested by customer, not qualifed to due no exact chip on
  sg site.
---
 drivers/mtd/ltq-spinand/ltq_spinand.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/drivers/mtd/ltq-spinand/ltq_spinand.c b/drivers/mtd/ltq-spinand/ltq_spinand.c
index 7c4dbee06786..76b9d11de9b5 100644
--- a/drivers/mtd/ltq-spinand/ltq_spinand.c
+++ b/drivers/mtd/ltq-spinand/ltq_spinand.c
@@ -158,6 +158,10 @@ static struct spinand_flash_dev spinand_flash_ids[] = {
 		{ .id = {0xc2, 0x22} }, 2048, 256, 0x20000,
 		 LP_OPTIONS | SPINAND_NEED_PLANE_SELECT, 2, 64},
 		},
+	{ {"SPI NAND MX35LF2G14AC 2Gbit",
+		{ .id = {0xc2, 0x20} }, 2048, 256, 0x20000,
+		 LP_OPTIONS | SPINAND_NEED_PLANE_SELECT, 2, 64},
+		},
 	{ {"SPI NAND 1Gbit 3,3v WN25N01GV",
 		{ .id = {0xef, 0xaa} }, 2048, 128, 0x20000, LP_OPTIONS, 2, 64},
 		},
